.travis.yml

```diff
@@ -18,17 +23,22 @@ install:
     conda config --add channels conda-forge --force
     conda config --set channel_priority strict
     conda config --set safety_checks disabled
+    conda create --name TEST python=$PY --file requirements.txt --file requirements_dev.txt --file requirements_docs.txt --file requirements_plus.txt
+    source activate TEST
+    conda info --all
 
-  # Install dependencies
-  - conda env create --file="${ENV_FILE}"
-  - source activate test
-  - conda list
-  - conda info --all
+install:
+  - pip install -e . --no-deps --force-reinstall
 
 script:
-  - python setup.py sdist >/dev/null
-  - python setup.py install
```

Comment:
The `tarball` test substitutes `python setup.py sdist` and `pip install -e . --no-deps --force-reinstall` is better then `python setup.py install` b/c it won't try to pull the dependencies and uses `pip`, as recommended by PyPA. (`python setup.py install` will be removed at some point.)